src/main/java/core/characters/CharacterManager.java

```diff
@@ -4,6 +4,9 @@
 
 // Need to import InputManager and ControlState
 
+/**
+ * TODO – the PlayerManager should inherit from here
+ */
 public class CharacterManager {
```

Comment:
Why would we have a parent? I think the use cases are distinct enough that we can have a single class for each and not worry about overlap

Reply:
maybe we don't need that – my thought is that PlayerManager could override certain methods like `openInventory` to actually display something onscreen before calling the super method to actually implement the functionality

my thinking seems weird to me so I think you're right

we just need a way to separate the usage of inventory and other systems from its presentation – NPCs need to use these systems, but only the Player needs it displayed